src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/TokenEndpointModule.cs

```diff
@@ -7,39 +7,19 @@
 
 namespace EdFi.DataManagementService.Frontend.AspNetCore.Modules;
 
-
 public class TokenEndpointModule : IEndpointModule
 {
     public void MapEndpoints(IEndpointRouteBuilder endpoints)
     {
         endpoints.MapPost("/oauth/token", GenerateToken);
     }
 
-    internal async Task GenerateToken(HttpContext httpContext, AuthenticationRequest authenticationRequest)
+    internal async Task GenerateToken(HttpContext httpContext)
```

Comment:
Bulk loader sends token requests in a strange (but legitimate) way. Instead of coding to accept that strange way, just ignore the incoming request.